src/test/java/org/commcare/formplayer/tests/MultiSelectCaseClaimTest.java

```diff
@@ -102,4 +102,9 @@ private void configureQueryMock() {
                 .thenReturn(FileUtils.getFile(this.getClass(),
                         "query_responses/case_claim_multi_select_response.xml"));
     }
+
+    @Override
+    protected boolean useCommCareArchiveReference() {
```

Comment:
no objections though this was [recently implemented](https://github.com/dimagi/formplayer/pull/1112/commits/575d7a7490c3c91b9b8e9ffb3133cc2a584228d4) and is not yet in `master`.  SInce multiselect work will take a while to merge, I can do a separate PR for this if you need these changes.

Reply:
nah, don't worry about it. We can do it on an as-needed basis as we update tests, or I'll work on it some late Friday afternoon.